packages/image_picker/image_picker_android/example/pubspec.yaml

```diff
@@ -20,7 +20,7 @@ dependencies:
     # the parent directory to use the current plugin's version.
     path: ../
   image_picker_platform_interface: ^2.10.0
-  mime: ^1.0.4
```

Comment:
Optional: All of the `example` constraints can just use `^2.0.0` unless there's a resolver failure; there's no ecosystem effect to having tighter constraints on the example. Only the `image_picker_for_web` constraint affects clients and thus needs to be a relaxed constraint.

Reply:
Ah yeah that's a good point, those are app targets. Done.